questions/tasks.py

```
@@ -255,3 +258,198 @@ def format_time_remaining(time_remaining: timedelta):
+    ).strftime("%d %B %Y %H:%M UTC")
+    formatted_timestep = formatted_timestep.lstrip("0")
+    if len(removed_options) == 1:
+        template = template.replace("Options ", "Option ", 1)
+        template = template.replace("Their", "Its", 1)
+    try:
+        text = template.format(
+            removed_options=removed_options_text,
+            timestep=formatted_timestep,
+            catch_all_option=catch_all_option,
+        )
+    except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)